docs/reference/cli.md

```diff
@@ -171,14 +171,14 @@ Options:
 
 ## init
 ```
-Usage: sqlmesh init [SQL_DIALECT] [OPTIONS]
+Usage: sqlmesh init [OPTIONS] [SQL_DIALECT]
 
-  Create a new SQLMesh repository. Argument SQL_DIALECT is required unless the dbt
-  template option is specified.
+  Create a new SQLMesh repository.
 
 Options:
-  -t, --template TEXT  Project template. Support values: airflow, dbt,
+  -t, --template TEXT  Project template. Supported values: airflow, dbt,
                        default.
+  -e, --empty          Create empty project directories.
```

Comment:
Rather than adding a new flag, can we just add another template with name `empty`?

Reply:
I like this more, didn't occur to me. Thanks!

Reply:
Ah on second thought @izeigerman, isn't generating an empty project orthogonal to the template? For example we might want _both_ and empty project _and_ and airflow template.

Reply:
We can yet another template for this if needed. Conversely, the `dbt` template doesn't generate any model files, and is technically `empty` as it is.

Reply:
The `dbt` template currently only creates the config file, so no SQLMesh project structure.

FWIW I'm ok either way, we can just roll out `--template empty` (current version of the PR) if it looks good.